lib/config/validation.ts

```diff
@@ -570,3 +571,41 @@ export async function validateConfig(
   warnings.sort(sortAll);
   return { errors, warnings };
 }
+
+export function isValidLogLevel(
```

Comment:
Move this to `lib/logger/utils.ts` otherwise we get circular dependencies.

Reply:
Where do you recommend the test to be put in `lib/logger/index.spec.ts` ?

Reply:
yes